server/game/cards/05-DT/OfficersBlaster.js

```diff
@@ -0,0 +1,48 @@
+const Card = require('../../Card.js');
+
+class OfficersBlaster extends Card {
+    //This creature gains +2 power.
+    //This creature gains, “Destroyed: Attach $this to this creature's right neighbor.”
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: [
+                ability.effects.modifyPower(2),
+                ability.effects.gainAbility('destroyed', {
+                    target: {
```

Comment:
there is not need for a target here. You can simply do:
gameAction: ability.actions.attach(() => ({
    target: context.source.controller.creaturesInPlay[context.source.controller.creaturesInPlay.indexOf(context.source) + 1]
    // note, using "this" here to get a pointer to the blaster
    upgrade: this
}))